learn/guides/organize-ballerina-code/package-references.md

```diff
@@ -32,6 +32,12 @@ redirect_from:
 ├── modules/
 │     ├── model/
 │     └── module1.test/
+├── documents/
```

Comment:
Shall we change this to have one directory and one file in the package root

Reply:
Updated to have,
├── documents/
├── sample.png